cypress/component/NotePreview.cy.tsx

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable cypress/no-async-tests */
+import { HideButton, PreviewButton } from "../../src/features/CourtCaseList/NotePreviewButton"
+describe("chevron icons check", () => {
+  it("shows the default down chevron with the label 'preview'", () => {
+    cy.mount(<PreviewButton />)
+    cy.get(".govuk-accordion-nav__chevron--down").should("exist")
```

Comment:
We could also check that `.govuk-accordion-nav__chevron` doesn't exist.

Reply:
both classes should exist, I have added an additional assertion to check that they both are present. Thanks.

Reply:
Thanks for explaining!